ammo-be/src/util/Config.ts

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable import/no-unresolved */
+// eslint-disable-next-line import/extensions
+import config from 'config/config.json';
+import packageJSON from 'package.json';
+import PackageJSON from 'types/PackageJSON';
```

Comment:
I do have troubles with the naming, why adding JSON ?

Reply:
'package' is a reserved keyword in JS, packageJSON seemed fair to me as it's importing the package.json